docs/html/topics/https-certificates.md

```diff
@@ -36,6 +36,8 @@ In order to use system trust stores, you need to use Python 3.10 or newer.
   Successfully installed SomePackage
   ```
 
+It's also possible to activate truststore persistently with `pip config set global.use-feature truststore`
+
```

Comment:
Done.

Reply:
I think this removes existing `use-feature` entries? I am not sure if we want to suggest this since it can potentially have unexpected side effects.

Reply:
> I think this removes existing use-feature entries? I am not sure if we want to suggest this since it can potentially have unexpected side effects.

Is it possible to instead _append_ existing configuration entries with `pip config ..`?

Reply:
I don’t think so. The feature is a bit marginal so no-one really cared enough to work on it.

Reply:
Ok. What about then adding some kind of disclaimer to the wording, like `Please be aware that this will replace any previously configured features.` ?

Reply:
It's worth remembering that `--use-feature` is for things that will be made the default in due course (and indeed that's the plan for truststore). Is it really worth adding this for a few releases, just to take it out again, when it's only documenting a special case of the general rule that all pip's command line options can be set in config (and via environment variables, for that matter)?

Reply:
It's up to the project maintainers to decide if it's worth making this change.

My motivation is that I'm working for a large corporation with a network infrastructure that depends on configuring pip to use `truststore`. The current wording on https://pip.pypa.io/en/stable/topics/https-certificates/ isn't especially helpful since it depends on manually adding `--use-feature=truststore` to every pip call. This isn't even feasible if using pip from the Visual Studio IDE. It would be more helpful if the documentation could be updated, so that I could point colleagues to clear and actionable advise on how to "fix" their pip setup once and for all without having to learn more about the pip configuration system.

Reply:
How about putting this in the "versionadded" note? Something like

> Experimental support, behind `--use-feature=truststore`.
> As with any other CLI option, this can be enabled globally if needed via config or environment variables.

That doesn't suggest there's anything special here, and makes it clear that the normal rules apply regarding config.

Reply:
Sounds reasonable. I've now tried to update the PR accordingly.